megaphone-validator/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lambda-megaphone-validator",
+  "version": "1.0.0",
+  "description": "An AWS lambda function that compares the file sizes of recently uploaded audio files with the original files",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arellaTV/lambda-megaphone-validator.git"
```

Comment:
Were you going to change this to the scpr github org? You may be on the hook for maintaining this forever if it's pointed to your github.

Reply:
@nakedsushi, Great catch! Yes, I was going to change this to SCPR's github, thanks for the reminder. Should be fixed in the latest commit 🙌